protocol/v2/qbft/instance/round_change.go

```diff
@@ -78,6 +83,9 @@ func (i *Instance) uponChangeRoundPartialQuorum(newRound specqbft.Round, instanc
 		return errors.Wrap(err, "failed to create round change message")
 	}
 
+	i.logger.Debug("got change round partial quorum, broadcasting change round message",
```

Comment:
We might want to log this every 10 rounds similar to timeout

Reply:
@amir-blox Modulo on `new_round` or `old_round? Problem is they both seem to so skip numbers so modulo isn't reliable, no?

Reply:
`newRound`, otherwise we see multiple logs in this flow..
it's fine that we won't see all the logs